Description: mask isSSE42() and isPCLMULQDQ() for ARM64 as well
 Otherwise defined but not used FTBFS will happen.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Forwarded: no
Last-Update: 2019-09-25

---

--- rocksdb-6.2.4.orig/util/crc32c.cc
+++ rocksdb-6.2.4/util/crc32c.cc
@@ -399,7 +399,7 @@ uint32_t ExtendImpl(uint32_t crc, const
 }
 
 // Detect if SS42 or not.
-#ifndef HAVE_POWER8
+#if !defined(HAVE_POWER8) && !defined(HAVE_ARM64_CRC)
 
 static bool isSSE42() {
 #ifndef HAVE_SSE42
@@ -435,7 +435,7 @@ static bool isPCLMULQDQ() {
 #endif
 }
 
-#endif  // HAVE_POWER8
+#endif  // HAVE_POWER8 or HAVE_ARM64_CRC
 
 typedef uint32_t (*Function)(uint32_t, const char*, size_t);
 
